source/common/http/header_map_impl.cc

```diff
@@ -369,14 +359,6 @@ void HeaderMapImpl::addReferenceKey(const LowerCaseString& key, absl::string_vie
 }
 
 void HeaderMapImpl::addCopy(const LowerCaseString& key, uint64_t value) {
-  auto* entry = getExistingInline(key.get());
-  if (entry != nullptr) {
-    char buf[32];
-    StringUtil::itoa(buf, sizeof(buf), value);
-    const uint64_t added_size = appendToHeader(entry->value(), buf);
```

Comment:
Wow! This is an amazing improvement. I was reviewing and left a comment:
> Sorry if I'm mistaking these, but are these `addCopy` behaviors now changed? It looks like we won't do comma concatenation for O(1) headers.

Only to know this must be covered by tests and realized that this is taken care of in `insertByKey` already. I remember this was so confusing before. Leaving this comment for posterity and gratitude. Thanks, Matt!!!